foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -27,82 +30,46 @@ Requires: /var/lib/tftpboot
-
+install -Dp -m0755 foreman-generate-bootloaders %{buildroot}%{_bindir}/foreman-generate-bootloaders
 
 %files
-%attr(755,root,root)
-%dir %{_datarootdir}/%{name}/grub
-%dir %{_datarootdir}/%{name}/grub2
-%attr(644,root,root)
-%{_datarootdir}/%{name}/grub/grubia32.efi
-%{_datarootdir}/%{name}/grub/grubx64.efi
-%{_datarootdir}/%{name}/grub/bootia32.efi
-%{_datarootdir}/%{name}/grub/bootx64.efi
-%{_datarootdir}/%{name}/grub2/grubaa64.efi
-%{_datarootdir}/%{name}/grub2/grubia32.efi
-%{_datarootdir}/%{name}/grub2/grubppc64.efi
-%{_datarootdir}/%{name}/grub2/grubppc64.elf
-%{_datarootdir}/%{name}/grub2/grubppc64le.efi
-%{_datarootdir}/%{name}/grub2/grubppc64le.elf
-%{_datarootdir}/%{name}/grub2/grubx64.efi
-%{_datarootdir}/%{name}/grub2/shimaa64.efi
-%{_datarootdir}/%{name}/grub2/shimaa64-redhat.efi
-%{_datarootdir}/%{name}/grub2/shimia32.efi
-%{_datarootdir}/%{name}/grub2/shimia32-redhat.efi
-%{_datarootdir}/%{name}/grub2/shimx64.efi
-%{_datarootdir}/%{name}/grub2/shimx64-redhat.efi
-
+%{_bindir}/foreman-generate-bootloaders
 
 %files tftpboot
-%ghost %{_sharedstatedir}/tftpboot/grub
 %ghost %{_sharedstatedir}/tftpboot/grub2
-%ghost %{_sharedstatedir}/tftpboot/grub/grubia32.efi
-%ghost %{_sharedstatedir}/tftpboot/grub/grubx64.efi
-%ghost %{_sharedstatedir}/tftpboot/grub/bootia32.efi
-%ghost %{_sharedstatedir}/tftpboot/grub/bootx64.efi
 %ghost %{_sharedstatedir}/tftpboot/grub2/grubaa64.efi
+%ghost %{_sharedstatedir}/tftpboot/grub2/grubia32.0
 %ghost %{_sharedstatedir}/tftpboot/grub2/grubia32.efi
 %ghost %{_sharedstatedir}/tftpboot/grub2/grubppc64.efi
 %ghost %{_sharedstatedir}/tftpboot/grub2/grubppc64.elf
 %ghost %{_sharedstatedir}/tftpboot/grub2/grubppc64le.efi
 %ghost %{_sharedstatedir}/tftpboot/grub2/grubppc64le.elf
+%ghost %{_sharedstatedir}/tftpboot/grub2/grubppc.efi
+%ghost %{_sharedstatedir}/tftpboot/grub2/grubppc.elf
 %ghost %{_sharedstatedir}/tftpboot/grub2/grubx64.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shimaa64.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shimaa64-redhat.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shimia32.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shimia32-redhat.efi
+%ghost %{_sharedstatedir}/tftpboot/grub2/shim.efi
 %ghost %{_sharedstatedir}/tftpboot/grub2/shimx64.efi
 %ghost %{_sharedstatedir}/tftpboot/grub2/shimx64-redhat.efi
+%ghost %{_sharedstatedir}/tftpboot/grub2/shimaa64.efi
+%ghost %{_sharedstatedir}/tftpboot/grub2/shimaa64-redhat.efi
 
 
 %post tftpboot
-# create copies (hardlinks) of files in tftpboot directory
-install -d -m0755 -o foreman-proxy %{buildroot}%{_sharedstatedir}/tftpboot/grub
-install -d -m0755 -o foreman-proxy %{buildroot}%{_sharedstatedir}/tftpboot/grub2
-cp -lpf %{_datarootdir}/%{name}/grub/* %{_sharedstatedir}/tftpboot/grub/
-cp -lpf %{_datarootdir}/%{name}/grub2/* %{_sharedstatedir}/tftpboot/grub2/
-
+%{_bindir}/foreman-generate-bootloaders x86 noinst
 
 
 %changelog
+* Wed Jan 24 2018 Lukas Zapletal <lzap+rpm@redhat.com> 201801241201-1
```

Comment:
We don't keep changelogs - tito extracts them from git log and then generates them.

Reply:
Removing.